src/components/assets/Account.vue

```diff
@@ -3,13 +3,12 @@
     <div class="container">
       <div class="row--details">
         <div class="column-account-name">
-          <au-icon
+          <!-- <au-icon
```

Comment:
@ayumitk @bobo-k2 
I think we don't need to comment out this. Just use the same code as [main](https://github.com/AstarNetwork/astar-apps/blob/main/src/components/assets/Account.vue#L6). What do you think?

Reply:
I commented this out because CI was failing otherwise. Let me uncomment and see what will happen.

Reply:
@impelcrypto fixed in https://github.com/AstarNetwork/astar-apps/pull/1001/commits/4c651d07be88409610e123d3f5a1c6b2dd403b9d